### PR TITLE
[Feature] mount google drive 

### PR DESCRIPTION
**Pull request by Skander-hadjromdhan**
new parameter in class gdrive that allows to mount google drive when set to True

### CODE REVIEW

**Comment by abhishekkrthakur** [comment]
Do kaggle kernels have google.colab package?

**Comment by Skander-hadjromdhan** [comment]
No, I just checked. I will update the code to handle this exception.

File: colabcode/code.py [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
-    def __init__(self, port=10000, password=None):
+    def __init__(self, port=10000, password=None, gcloud=False):
         self.port = port
         self.password = password
+        self.gcloud = gcloud
         self._install_code()
         self._start_server()
         self._run_code()
-
+
```
Comment (abhishekkrthakur) [inline colabcode/code.py:15]: remove extra spaces please :)

File: colabcode/code.py [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
         self._run_code()
-
+        
     def _install_code(self):
         subprocess.run(
             ["wget", "https://code-server.dev/install.sh"], stdout=subprocess.PIPE
         )
         subprocess.run(["sh", "install.sh"], stdout=subprocess.PIPE)
-
+
```
Comment (abhishekkrthakur) [inline colabcode/code.py:21]: remove extra spaces please :)

File: colabcode/code.py [hunk 3]
```diff
@@ -1,32 +1,36 @@
...
+        
     def _start_server(self):
         active_tunnels = ngrok.get_tunnels()
         for tunnel in active_tunnels:
             public_url = tunnel.public_url
             ngrok.disconnect(public_url)
         url = ngrok.connect(port=self.port)
         print(f"Code Server can be accessed on: {url}")
-
+
```
Comment (abhishekkrthakur) [inline colabcode/code.py:29]: remove extra spaces please :)

File: colabcode/code.py [hunk 4]
```diff
@@ -1,32 +1,36 @@
...
         for tunnel in active_tunnels:
             public_url = tunnel.public_url
             ngrok.disconnect(public_url)
         url = ngrok.connect(port=self.port)
         print(f"Code Server can be accessed on: {url}")
-
+        
     def _run_code(self):
         os.system(f"fuser -n tcp -k {self.port}")
+        if self.gcloud:
```
Comment (abhishekkrthakur) [inline colabcode/code.py:32]: What about a `_mount` function instead?

File: colabcode/code.py [hunk 5]
```diff
@@ -1,32 +1,36 @@
 import os
 import subprocess
 from pyngrok import ngrok
+from google.colab import drive
```
Comment (abhishekkrthakur) [inline colabcode/code.py:4]: try, except ImportError.....

**Review by abhishekkrthakur — CHANGES_REQUESTED** [review]

**Review by Skander-hadjromdhan — COMMENTED** [review]
done

**Review by Skander-hadjromdhan — COMMENTED** [review]
Exception changed

**Review by abhishekkrthakur — APPROVED** [review]

**Pull request merged**